<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelGraphics</class>
 <widget class="QWidget" name="PanelGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="labelGraphics">
       <property name="text">
        <string>Graphics:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetGraphics" name="frameGraphics" native="true"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGraphics"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetGraphics</class>
   <extends>QWidget</extends>
   <header>widgetgraphics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
